spec/system/task_spec.rb

```diff
@@ -3,34 +3,34 @@
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
     context '正常系' do
+      let(:project) { create(:project) }
+      let(:task) { create(:task) }
```

Comment:
describe全体で呼び出せる様に、1つ上の階層で定義すると、削除時以外のletを記載する必要が無くなりますね。
```suggestion
RSpec.describe 'Task', type: :system do
  let(:project) { create(:project) }
  let(:task) { create(:task) }
  
  describe 'Task一覧' do
    context '正常系' do
```

Reply:
なるほど！95da3a739ff7da3483d6197b8c6558decbfcabc7 で修正しました。

Reply:
95da3a7 で確認しました。